rs/ethereum/cketh/test_utils/src/lib.rs

```diff
@@ -581,6 +580,70 @@ impl CkEthSetup {
         }
         self
     }
+
+    pub fn max_logs_block_range(&self) -> u64 {
+        if self.evm_rpc_id.is_none() {
+            799
+        } else {
+            500
+        }
+    }
+
+    pub fn received_eth_event_topic(&self) -> serde_json::Value {
```

Comment:
Isn't there quite a bit of code duplication here?

Reply:
Yes!

Reply:
I introduced a little helper method `json_topic` to take care of this.